.theia/settings.json

```diff
@@ -1,6 +1,5 @@
 {
-  "editor.formatOnSave": true,
```

Comment:
Accidental commit?

Reply:
Probably in this one, but not in general: https://github.com/eclipse-theia/theia/issues/13475. We should be able to use Theia to develop Theia.